lib/github_service/issue_mover.rb

```diff
@@ -0,0 +1,46 @@
+module GithubService
+  class IssueMover
+    Error = Class.new(StandardError)
+
+    attr_reader :old_issue, :new_issue, :fq_repo_name
+
+    def initialize(issue, fq_repo_name)
+      @old_issue = issue
+      @fq_repo_name = fq_repo_name
+      @organization_name, @repo_name = fq_repo_name.split("/")
+    end
+
+    def execute
+      validate
+
+      new_issue = GithubService.create_issue(fq_repo_name, old_issue.title, new_issue_body)
+      GithubService.add_comment(old_issue.fq_repo_name, old_issue.number, "I've moved this issue to #{new_issue.html_url}")
+      GithubService.close_issue(old_issue.fq_repo_name, old_issue.number)
+    end
+
+    private
+
+    def validate
+      error = if old_issue.repo_name.downcase == @repo_name.downcase
+                "Issue already exists on the '#{old_issue.repo_name}' repo."
+              elsif old_issue.pull_request?
+                "You can't move a pull request!"
+              elsif old_issue.organization_name.downcase != @organization_name.downcase
+                "Cannot move issue to repo outside of the #{old_issue.organization_name} organization."
+              elsif !GithubService.repository?(fq_repo_name)
+                "Repository does not exist or is unreachable."
+              end
+      raise Error, error if error
+    end
+
+    def new_issue_body
+      <<-EOS
+#{old_issue.body}
+
+---
+
+*This issue was moved to this repository from #{old_issue.html_url}, originally opened by @#{old_issue.author}*
```

Comment:
Do you happen to have a screenshot of this?  Just from reading it, it looks nice, but would like to see